omniscidb/QueryEngine/Compiler/Backend.h

```diff
@@ -90,6 +91,8 @@ class CPUBackend : public Backend {
 
   CodegenTraits traits() const { return CodegenTraits::get(0, 0, llvm::CallingConv::C); };
 
+  void setSharedMemory(bool is_gpu_smem_used){};
```

Comment:
Might be better to enforce some semantics here if we expect `is_gpu_smem_used_` to always be false for the CPU backend.